spec/unit/provider/service/launchd_spec.rb

```diff
@@ -65,7 +75,7 @@
   end
 
   [[10, '10.6'], [13, '10.9']].each do |kernel, version|
-    describe "when checking whether the service is enabled on OS X #{version}" do
+    describe "when checking whether the service is enabled on OS X #{version}", :unless => Puppet.features.microsoft_windows? do
```

Comment:
Since the behavior here is mostly mocked / stubbed... is it best to guard, or best to fix any mocking within Windows?